cmd/dex/serve.go

```diff
@@ -145,6 +149,8 @@ func serve(cmd *cobra.Command, args []string) error {
 		s = storage.WithStaticPasswords(s, passwords)
 	}
 
+	serverLogger := logger.WithField("unit", "server")
```

Comment:
maybe we can annotate connectors for now. E.g. `logger.WithField("connector", "ldap")` since we'll only have one storage I don't think we'll need it